docs/griptape-cloud/structures/run-structure.md

```diff
@@ -1,21 +1,23 @@
 # Running a Structure
 
-Once your Structure is created and deployed, you can run your Structure one of three ways outlined below. You view the output of any of your runs, no matter how you created them, in the `Runs` tab of your Structure.
+Once your Structure is created and deployed, you can run your Structure one of three ways outlined below. You may view the details of any of your runs, no matter how you created them, in the `Runs` tab of your Structure.
+
+To learn more about Structure Run details and output, look at the [Structure Run Events](./structure-run-events.md) documentations.
 
 ## From the Cloud Console
 
 In the cloud console, click on the name of the Structure you wish to run and then go to the `Test` tab. Here you can specify arguments to pass to your Structure run and any run-specific environment variables you need.
 
 When passing arguments through the cloud console, pass each new argument on a new line. For example if your local code is ran with the inputs `-i input_file.txt` then the arguments you would pass in the cloud would be:
 
-```
+```shell
```

Comment:
I've seen ```bash used elsewhere, do you know what the difference is?

Reply:
bash is just a specific type of shell. in this example it makes no difference

Reply:
I'm not sure, but it seems to render the same (at least for this snippet), and bash is shorter so I'll switch to that!